config/initializers/time_formats.rb

```diff
@@ -0,0 +1,3 @@
+# Default format for displaying dates and times
+Date::DATE_FORMATS[:default] = '%d/%m/%Y'
```

Comment:
What's this for? We actually use `%-d %B %Y` or something in most places, if I recall correctly. And shouldn't the Time one have a time in it?

Reply:
I will remove it

Reply:
it does who dates like this without it
![image](https://user-images.githubusercontent.com/63246336/80596193-e3dfc880-8a1d-11ea-8dad-0b4627a4419b.png)

Reply:
"%-d %B %Y" is used in plain prints, here I am doing a input text this is the main diff